src/parser.js

```diff
@@ -252,6 +255,16 @@ function processCategoryTags(post, domain) {
 		.map(({ $: attributes }) => decodeURIComponent(attributes.nicename));
 }
 
+function getDescription(post){
+	const postmeta = post.postmeta.find(postmeta => postmeta.meta_key[0] === '_yoast_wpseo_metadesc');
+	if(postmeta === undefined){
+		return " ";
```

Comment:
```suggestion
		return "";
```

Omit this to make filtering more easy later on.